Makefile.defs

```diff
@@ -19,7 +19,7 @@ NDK?=$(TOOLCHAIN_DIR)/android-ndk-r15c
 NDKABI?=14
 
 # Detect LuaRocks 3
-LR_VER_NUM := $(shell luarocks --version | grep -o [0-9].[0-9])
+LR_VER_NUM := $(shell luarocks --version | grep -o [0-9].[0-9]\$)
```

Comment:
This isn't right.

It's meant to do something like:

2.4.2 → 2.4
3.8.0 → 3.8

This way that becomes:
2.4.2 → 4.2
3.8.0 → 8.0

I'm not immediately sure otoh what the best way to work around this would be. Perhaps we need awk instead of grep.

But given your OP, it might suffice to do something fairly dumb looking, like:
```
luarocks --version | grep -o " [0-9].[0-9]" | grep -o "[0-9].[0-9]"
```
(Or alternatively with -P to be able to use non-matching groups, though I doubt that'd be more legible.)